Formula/circleci.rb

```diff
@@ -3,8 +3,8 @@ class Circleci < Formula
   homepage "https://circleci.com/docs/2.0/local-cli/"
   # Updates should be pushed no more frequently than once per week.
```

Comment:
Updated 2 days ago:
https://github.com/Homebrew/homebrew-core/commit/edcfef1c296ccc247a8b2c880a072f339e40f10a

Reply:
@dawidd6 Ahh, understood.

Without knowing the full extent of Homebrew's contribution policy, I'd like to clarify that this is a hotfix that addresses an issue currently breaking all functions of the CircleCI CLI; this version bump makes the app useable again. If there's a way you can consider this an exceptional case, we'd be eternally grateful.

Otherwise I'll try again in a few days. Thanks!